assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -785,39 +784,6 @@ form {
   text-decoration: underline;
 }
 
-.form__submit {
```

Comment:
These style rules are moved not removed. As they are shared across landing and thank you pages they've been put in a new section.